### PR TITLE
Experimental RainbowFit in Python

### PR DESCRIPTION
**Pull request by hombit**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,11 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
---
+- **Breaking change in experimental features** Multiband support is introduced for features implemented in Python. It changes class inheritance interface in a backward-incompatible way
+- `light-curve[full]` extras which installs all optional Python dependencies required by experimental features
+- Experimental `RainbowFit` feature for fitting multiband light curves with a single model, Russiel+23 in prep. It requires Python 3.8 or later because of `iminuit` dependency
```
Comment (erusseil) [inline CHANGELOG.md:14]: "Russiel" should be corrected to Russeil

File: light-curve/light_curve/light_curve_py/features/rainbow.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            "reference_time": (t[0] - 10 * t_amplitude, t[-1] + 10 * t_amplitude),
+            "amplitude": (0.0, 10 * m_amplitude),
+            "rise_time": (0.0, 10 * t_amplitude),
+            "fall_time": (0.0, 10 * t_amplitude),
+            "Tmin": (1e2, 1e6),  # K
+            "delta_T": (0.0, 1e6),  # K
+            "k_sig": (0.0, 10 * t_amplitude),
+        }
+        if self.with_baseline:
+            initial_guesses.update({f"baseline_{band}": 0.0 for band in self.band_wave_cm})
```
Comment (erusseil) [inline light-curve/light_curve/light_curve_py/features/rainbow.py:183]: Shouldn't you put np.min(m) instead of 0 for the initial guess of baseline ?

**Review by erusseil — COMMENTED** [review]
Overall it looks amazing ! Thank you for the implementation

File: light-curve/light_curve/light_curve_py/features/rainbow.py (hunk 2 above)
Comment (hombit) [reply light-curve/light_curve/light_curve_py/features/rainbow.py]: Thanks for this. There actually was an error in a different part, I've forgotten to implement flux normalization for the baseline-full case. It should be fixed now

**Review by erusseil — APPROVED** [review]
Indeed good catch on the baseline error. Now I think everything is correct

**Review by anlava — APPROVED** [review]

**Pull request merged**